priv/repo/datasets.json

```diff
@@ -35,7 +35,7 @@
 }, {
     "spatial": "Communauté d'agglomération Royan Atlantique",
     "coordinates": [-1.027697, 45.625611],
-    "slug": "donnees-gtfs-du-reseau-de-transport-public-carabus-1",
```

Comment:
Why this commit? The old slug seems to be the correct one:

http://next.data.gouv.fr/fr/datasets/donnees-gtfs-du-reseau-de-transport-public-carabus-1/

Reply:
Forget what I said. I was on next… :man_facepalming: